clang/lib/3C/RewriteUtils.cpp

```diff
@@ -432,6 +434,7 @@ void RewriteConsumer::HandleTranslationUnit(ASTContext &Context) {
     // Cast placement must happen after type expression rewriting (i.e. cast and
     // compound literal) so that casts to unchecked pointer on itype function
     // calls can override rewritings of casts to checked types.
+    CLV.TraverseDecl(D);
```

Comment:
Order matters, I presume? If so, say that `CLV` must go first. But, per my comment above, maybe you don't need `ECPV` to refer to `CLV` directly, but instead can pass in the generated set ?